packages/mobile/src/identity/privateHashing.ts

```diff
@@ -85,6 +92,17 @@ function* doFetchPhoneHashPrivate(e164Number: string) {
   return details
 }
 
+export function* getSaltCache(e164Number: string) {
+  const saltCache: E164NumberToSaltType = yield select(e164NumberToSaltSelector)
+  return saltCache[e164Number]
+}
+
+export function* balanceSufficientForQuotaRetrieval() {
```

Comment:
Duped with the one in CK now right?

Reply:
This one is a saga and the one in CK is a pure function. `balanceSufficientForQuotaRetrieval` is a client of `isSufficientBalanceForQuotaRetrieval`